src/app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class LoginController extends Controller
+{
+    public function index()
```

Comment:
return type

Reply:
I already resolved this comment, sir. Thank you po!